filters/subunit-filter

```diff
@@ -91,6 +91,9 @@ def make_options(description):
     parser.add_option("-F", "--only-genuine-failures", action="callback",
         callback=only_genuine_failures_callback,
         help="Only pass through failures and exceptions.")
+    parser.add_option("--rename", action="append", nargs=2,
+        help="Apply the specified regex to test names.",
```

Comment:
"Apply specified regex substitutions to test names"

Reply:
Done.